src/stdlib/io.rs

```diff
@@ -9,19 +9,34 @@ use std::{io::Write, ops::Deref};
 fndr_native_func!(
     /// Prints to stdout
     print_func,
-    |_, item| {
+    |_, item, argv| {
         print!("{}", item.to_string());
-        let mut lock = std::io::stdout().lock();
-        let _ = lock.flush();
+        let argv = match &*argv {
+            List(l) => l.deref(),
+            _ => unreachable!(),
+        };
+        for item in argv {
+            print!("{}", item.to_string());
+        }
         Ok(item)
     }
 );
 
 fndr_native_func!(
     /// Prints to stdout and adds a newline
     println_func,
-    |_, item| {
-        println!("{}", item.to_string());
+    |_, item, argv| {
+        print!("{}", item.to_string());
+        let argv = match &*argv {
+            List(l) => l.deref(),
+
+            e => unreachable!("{:?}", e),
```

Comment:
Can we add a message to this? Currently looks like it just prints whatever the value was, it would help to include that it was supposed to be a list